libs/walproposer/src/api_bindings.rs

```diff
@@ -450,6 +450,9 @@ pub fn empty_shmem() -> crate::bindings::WalproposerShmemState {
         replica_promote: false,
         min_ps_feedback: empty_feedback,
         wal_rate_limiter: empty_wal_rate_limiter,
+        num_safekeepers: 0,
```

Comment:
not sure if this is a safe change wrt compatibility -- need someone from compute to review